int64[] destination_id
float64[] distance
time[] stamp
